### ISSUE TITLE
Unicode error when having unicode character in folder path

### ISSUE THREAD

**Issue by htp84** [linkedin/shiv#110]
Hi, thank you for making an excellent project like this open-source. I have a Unicode character in my folder path and therefore get the following error:

```python
Traceback (most recent call last):
  File "c:\programdata\miniconda3\envs\energi_xl\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\programdata\miniconda3\envs\energi_xl\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\ProgramData\Miniconda3\envs\energi_xl\Scripts\shiv.exe\__main__.py", line 9, in <module>
  File "c:\programdata\miniconda3\envs\energi_xl\lib\site-packages\click\core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "c:\programdata\miniconda3\envs\energi_xl\lib\site-packages\click\core.py", line 697, in main
    rv = self.invoke(ctx)
  File "c:\programdata\miniconda3\envs\energi_xl\lib\site-packages\click\core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "c:\programdata\miniconda3\envs\energi_xl\lib\site-packages\click\core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "c:\programdata\miniconda3\envs\energi_xl\lib\site-packages\shiv\cli.py", line 169, in main
    pip.install(["--target", str(tmp_site_packages)] + list(pip_args))
  File "c:\programdata\miniconda3\envs\energi_xl\lib\site-packages\shiv\pip.py", line 65, in install
    click.echo(output.decode().rstrip())
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xf6 in position 15: invalid start byte
```
I fixed it by modifying the *install* function in *shiv/src/shiv/pip.py*. 
I added the argument *text=True* to *subprocess.Popen* and removed *decode()* from *click.echo(output.decode().rstrip())* (line 65).

i.e. from
```python
...
    process = subprocess.Popen(
        [sys.executable, "-m", "pip", "--disable-pip-version-check", "install", *args],
        stdout=subprocess.PIPE,
        stderr=subprocess.STDOUT,
        env=subprocess_env,
    )

for output in process.stdout:
    if output:
        click.echo(output.decode().rstrip())
...
```
to

```python
...
    process = subprocess.Popen(
        [sys.executable, "-m", "pip", "--disable-pip-version-check", "install", *args],
	stdout=subprocess.PIPE,
	stderr=subprocess.STDOUT,
	env=subprocess_env,
	text=True,
	)

for output in process.stdout:
    if output:
	click.echo(output.rstrip())
...
```
I do not know if this is the best, or even, a good solution but in my case it gets the job done.

**Comment by lorencarvalho** [comment]
That *is* a good solution! I would happily merge a PR with this change if you'd like to make one, otherwise I could open one :)

**Comment by htp84** [comment]
Nice, I gladly submit a PR.

**Comment by lorencarvalho** [comment]
thanks for the PR!

merged & released to pypi @ 0.0.50
